app/models/helpers/helpers.rb

```diff
@@ -0,0 +1,13 @@
+module Helpers
```

Comment:
Yes, I see your point. 
I intentionally created `helpers.rb` so people to dump all helpers there. Otherwise we will have a bunch of `.rb` files with only one method in it and that will make rails load even slower.